Main/Source/iconf.cpp

```diff
@@ -162,6 +164,10 @@ truthoption ivanconfig::AllWeightIsRelevant("AllWeightIsRelevant",
                                           "Only pile items with equal weight on lists", //clutter are useful now for crafting so their weight matters...
                                           "",
                                           false);
+truthoption ivanconfig::DropBeforeOffering("DropBeforeOffering",
+                                          "Drop the item on altar before offering it.",
+                                          "Beware it may be owned floor!",
```

Comment:
Maybe add "Automatically drop offered items on an altar to prevent them from cluttering your inventory should the god not accept your gift." I think the players will like to know what do such options actually do. :)

Reply:
that end up being also the code flow :>